apis/mattermost/v1alpha1/clusterinstallation_sizes.go

```diff
@@ -60,16 +61,20 @@ var size100 = ClusterInstallationSize{
 // CloudSize10String represents estimated Mattermost Cloud installation sizing for 10 users.
 const CloudSize10String = "cloud10users"
 
+
+var cpuInstallationRequests10u = getEnv("MM10_CPU_REQUESTS", "10m")
+var memoryInstallationRequests10u = getEnv("MM10_MEM_REQUESTS", "150Mi")
```

Comment:
I would suggest the naming to be 
```suggestion
var cpuInstallationRequests10u = getEnv("MM_CPU_REQUESTS_10", "10m")
var memoryInstallationRequests10u = getEnv("MM_MEM_REQUESTS_10", "150Mi")
```
As firstly when I saw I thought it was for licensing purpose like the E10 or E20

Reply:
@angeloskyratzakos Make sense thanks for the comment.

Reply:
Also why to have the number within the variable as in the future if we change the `MM_CPU_REQUESTS_10` to be `20m` it wouldn't make sense

Reply:
@angeloskyratzakos Is a default value when ENV is not set check the helper function added to the bottom

Reply:
maybe something like `MM_SIZE10_CPU_REQUESTS`

Reply:
that's even better